libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -33,12 +33,15 @@
 
 extern const AP_HAL::HAL& hal;
 
-ObjectArray<mavlink_statustext_t> AP_Frsky_Telem::_statustext_queue(FRSKY_TELEM_PAYLOAD_STATUS_CAPACITY);
+AP_Frsky_Telem::AP_Frsky_Telem(void) :
```

Comment:
I wonder... could we eliminate this constructor altogether by using brace-initialisation of _statustext_queue within the header file?

Happy to merge this and do that as a future PR...

Reply:
I messed around a bit and got the syntax wrong, so I moved it out cause I knew the syntax for that :) Will come back to trying with braces.